src/styles/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product {
+  width: 200px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid white;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 19px 0;
```

Comment:
use only margin-bottom to make an outer vertical indent between elements
Also you can make your image as `block` and add `margin: 0 auto` to place at the center